### PR TITLE
Add Google Playstore Description

### PR DESCRIPTION
**Pull request by superpoussin22**
### Describe the Pull Request

```
Add Google Plastore Description
Add Privacy Link
```

### Checklist for PR

- [X] Run MobSF unit tests and lint `tox -e lint,test`.
- [X] Tested Working on Linux, Mac, Windows, and Docker
- [X] Make sure build is passing on your PR. [Build Status](https://travis-ci.com/MobSF/Mobile-Security-Framework-MobSF/pull_requests)

### CODE REVIEW

File: StaticAnalyzer/views/android/playstore.py [hunk 1]
```diff
@@ -13,6 +13,9 @@ def get_app_details(package_id):
         det = app(package_id)
         det.pop('descriptionHTML', None)
         det.pop('comments', None)
+        description = det['description'].replace('<b>', '')
```
Comment (ajinabraham) [inline StaticAnalyzer/views/android/playstore.py]: @superpoussin22 Sorry for confusing., I meant `</br>` with `\n` 
`<b>` and `</b>` with ``

Comment (ajinabraham) [reply StaticAnalyzer/views/android/playstore.py]: Can be shortened

   det['description'] = det['description'].replace('<b>', '').replace('</b>', '').replace('<br>', '\n').replace('</br>', '\n')

**Comment by ajinabraham** [comment]
@superpoussin22  or a neat way is
```
from bs4 import BeautifulSoup

html_str = '''
<td><a href="http://www.fakewebsite.com">Please can you strip me?</a>
<br/><a href="http://www.fakewebsite.com">I am waiting....</a>
</td>
'''
soup = BeautifulSoup(html_str)

print(soup.get_text()) 
#or via attribute of Soup Object: print(soup.text)
```

Do this if we have beautifulsoup already included with MobSF

Alternatively
```
import re

TAG_RE = re.compile(r'<[^>]+>')

def remove_tags(text):
    return TAG_RE.sub('', text)
```

File: StaticAnalyzer/views/android/playstore.py (hunk 1 above)
Comment (superpoussin22) [reply StaticAnalyzer/views/android/playstore.py]: will do

**Comment by superpoussin22** [comment]
will move to BeautifulSoup

**Pull request merged**